.github/ISSUE_TEMPLATE/pubdev_checklist.md

```diff
@@ -0,0 +1,13 @@
+Referece Documents:
+
+ [Pub.Dev List](https://docs.google.com/document/d/1GpHSJzQHTqzOpQlObOODWXloHoQRjy2-B1TRT-PMVlg/edit#)
```

Comment:
Better to just have the contents of the document here than send people on a treasure hunt.

Reply:
Will do.  List added below.